clients/src/test/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventProcessorTest.java

```diff
@@ -183,6 +186,38 @@ public void testAssignmentChangeEventWithException() {
         assertInstanceOf(IllegalStateException.class, e.getCause());
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
```

Comment:
do we get any value from these params? I expect that having or not a group ID does not change the course of this test that is only about seeking to a position

Reply:
Yes, I think we don't need to test both situations for this case. Thanks.